packages/zoe/src/zoeService/utils.d.ts

```diff
@@ -13,22 +14,24 @@ type ContractFacet<T extends {} = {}> = {
 /**
  * Installation of a contract, typed by its start function.
  */
-declare const StartFunction: unique symbol;
```

Comment:
ok, so no more unique symbol... though we take on responsibility for any collisions for strings such as `StartFunction`.